src/server/routes/index.js

```diff
@@ -53,19 +53,6 @@ exports.player = async function player (req, res, next) {
       },
       embedUrl: function () {
         return 'https://portal.paratii.video/embed/' + video._id
-      },
-      meta: function () {
-        return meta
-      },
-      embed: function () {
-        return path === '/embed/:id'
```

Comment:
This was the culprit. We needed to pass this directly as a boolean, not as a render helper